src/hm-admin/hm-admin.component.ts

```diff
@@ -1,22 +1,25 @@
 import {Component, Input, OnInit} from '@angular/core';
+import {HTTP_PROVIDERS} from '@angular/http';
 import {ConfigService} from './services/config/config.service';
+import {CoreDefinition} from './services/core/CoreDefinition.service';
 
 @Component({
   moduleId: module.id,
   selector: 'hm-admin-app',
   templateUrl: 'hm-admin.component.html',
   styleUrls: ['hm-admin.component.css'],
-  providers: [ConfigService]
+  providers: [ConfigService, CoreDefinition, HTTP_PROVIDERS]
 })
 export class HmAdminComponent implements OnInit {
   @Input('config')
   config: any;
 
-
-  constructor(private configService:ConfigService) {}
+  constructor(private configService: ConfigService, private coreDef: CoreDefinition) {}
```

Comment:
once you suffix with Service and the second time not. Keep consistency and try to avoid abbreviation in variable names